release/yum-repo/Earthfile

```diff
@@ -10,6 +10,7 @@ rpm:
     ARG EARTHLY_VERSION=$(echo "$RELEASE_TAG" | cut -c 2-)
     FROM +deps
     WORKDIR /work
+    RUN test ! -z "$EARTHLY_PLATFORM" || (echo "EARTHLY_PLATFORM is required" && exit 1)
```

Comment:
This is set during the release process right, I dont need to provide this during the release?

Reply:
it is. I just added this as I was calling the `+rpm` target directly and the error message when this is missing isn't as clear.